nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/aggregate/HistogramAggregate.java

```diff
@@ -139,6 +139,6 @@ protected int parseInt(String key) {
         } else if ("+Infinity".equals(key)) {
             return Integer.MAX_VALUE;
         }
-        return Integer.parseInt(key);
+        return Math.round(Float.parseFloat(key));
```

Comment:
Note that `Math.round` rounds to nearest, if that matters. I don't know the usage of this code but is rounding actually needed? `BucketRange` takes a `Number` and internally uses a `Double` anyway.

Reply:
may be, here I just didn't want to change the logic and support a key expressed as float